<ui version="4.0" >
 <class>CrystalWidget</class>
 <widget class="QWidget" name="CrystalWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>115</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>0</y>
     <width>301</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title" >
    <string>Current Crystal</string>
   </property>
   <widget class="QLabel" name="OrientationLine1Label" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>X-axis</string>
    </property>
   </widget>
   <widget class="QLabel" name="OrientationLine1Label_2" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>50</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>Y-axis</string>
    </property>
   </widget>
   <widget class="QLabel" name="OrientationLine1Label_3" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>70</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>Z-axis</string>
    </property>
   </widget>
   <widget class="QLabel" name="label10" >
    <property name="geometry" >
     <rect>
      <x>90</x>
      <y>30</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>[</string>
    </property>
   </widget>
   <widget class="QLabel" name="label10_2" >
    <property name="geometry" >
     <rect>
      <x>90</x>
      <y>50</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>[</string>
    </property>
   </widget>
   <widget class="QLabel" name="label10_3" >
    <property name="geometry" >
     <rect>
      <x>90</x>
      <y>70</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>[</string>
    </property>
   </widget>
   <widget class="QLabel" name="OrientationLine1_1" >
    <property name="geometry" >
     <rect>
      <x>100</x>
      <y>30</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="layoutDirection" >
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text" >
     <string>-99.999</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="OrientationLine2_1" >
    <property name="geometry" >
     <rect>
      <x>100</x>
      <y>50</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="layoutDirection" >
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text" >
     <string>-9.999</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="OrientationLine3_1" >
    <property name="geometry" >
     <rect>
      <x>100</x>
      <y>70</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="layoutDirection" >
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text" >
     <string>-9.999</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="OrientationLine1_2" >
    <property name="geometry" >
     <rect>
      <x>160</x>
      <y>30</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="layoutDirection" >
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text" >
     <string>-99.999</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="OrientationLine2_2" >
    <property name="geometry" >
     <rect>
      <x>160</x>
      <y>50</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="layoutDirection" >
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text" >
     <string>-9.999</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="OrientationLine3_2" >
    <property name="geometry" >
     <rect>
      <x>160</x>
      <y>70</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="layoutDirection" >
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text" >
     <string>-9.999</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="OrientationLine1_3" >
    <property name="geometry" >
     <rect>
      <x>220</x>
      <y>30</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="layoutDirection" >
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text" >
     <string>-99.999</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="OrientationLine2_3" >
    <property name="geometry" >
     <rect>
      <x>220</x>
      <y>50</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="layoutDirection" >
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text" >
     <string>-9.999</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="OrientationLine3_3" >
    <property name="geometry" >
     <rect>
      <x>220</x>
      <y>70</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="layoutDirection" >
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text" >
     <string>-9.999</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label11" >
    <property name="geometry" >
     <rect>
      <x>280</x>
      <y>30</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label11_2" >
    <property name="geometry" >
     <rect>
      <x>280</x>
      <y>50</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label11_3" >
    <property name="geometry" >
     <rect>
      <x>280</x>
      <y>70</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>]</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
